docs/magento2/search-engine.mdx

```diff
@@ -160,6 +160,14 @@ attribute and as a facet:
 You can then run the indexer so that the products are indexed in Algolia's
 index.
 
+:::note
+
+`category_ids` is the only required facets. Depending on your project, you can
```

Comment:
@dpobel shouldn't `price` facet be required as well?

Reply:
it's not required as far as I can remember